airflow/providers/postgres/hooks/postgres.py

```diff
@@ -53,13 +58,13 @@ class PostgresHook(DbApiHook):
     default_conn_name = 'postgres_default'
     supports_autocommit = True
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.schema = kwargs.pop("schema", None)
         self.connection = kwargs.pop("connection", None)
         self.conn = None
```

Comment:
Hi, could someone tell me please, what's the diff between this tow line of code (two connections)
are both same connection ? I assume no

``` python
        self.connection = kwargs.pop("connection", None)
        self.conn = None
```

Reply:
Taking a look at the hook logic it seems that:
- `self.connection` is an Airflow `Connection` class so it provides information on what parameters should be used for establishing postgres connection
- `self.conn` is direct connection to postgres: 
https://github.com/apache/airflow/blob/12a652f5344c7f03c3d780556ca1829b235fdb2d/airflow/providers/postgres/hooks/postgres.py#L104

Reply:
This Airflow Connection come from this class I assume `from airflow.models import Connection`
 
so can we make this change :

```diff
from airflow.models import Connection

- self.connection = kwargs.pop("connection", None)
+ self.connection: Optional[Connection] = kwargs.pop("connection", None)
```

Reply:
Yes 👌